mailbox/api/src/main/java/org/apache/james/mailbox/model/Attachment.java

```diff
@@ -107,6 +107,10 @@ public InputStream getStream() throws IOException {
         return new ByteArrayInputStream(bytes);
     }
 
+    public byte[] getBytes() {
+        return Arrays.copyOf(bytes, bytes.length);
```

Comment:
I am not sure if the copy is overkill. Should we risk that someone mutate the object to save RAM?